MoltenVKShaderConverter/MoltenVKShaderConverter/SPIRVReflection.h

```diff
@@ -173,6 +173,42 @@ namespace mvk {
 #endif
 	}
 
+	auto addSat = [](uint32_t a, uint32_t b) { return a == uint32_t(-1) ? a : a + b; };
+
+	template<typename Vo>
+	static inline uint32_t getShaderOutputStructMembers(const SPIRV_CROSS_NAMESPACE::CompilerReflection& reflect, Vo& outputs,
+														const SPIRV_CROSS_NAMESPACE::SPIRType* structType, spv::StorageClass storage,
+														bool patch, uint32_t loc) {
+		bool isUsed = true;
+		auto biType = spv::BuiltInMax;
+		size_t mbrCnt = structType->member_types.size();
+		for (uint32_t mbrIdx = 0; mbrIdx < mbrCnt; mbrIdx++) {
+			// Each member may have a location decoration. If not, each member
+			// gets an incrementing location based the base location for the struct.
```

Comment:
```suggestion
			// gets an incrementing location based on the base location for the struct.
```
I know that was there in the original, but this seems like a good time to fix the typo.

Reply:
Done.